### ISSUE TITLE
[Feature request]: able to set all movies to 00:00 at once

### ISSUE THREAD

**Issue by the-liquid-metal** [vzhd1701/gridplayer#20]
### Suggestion

As the title says.

**Comment by vzhd1701** [comment]
Do you mean that they should have an ability to stop upon reaching the end without looping?

**Comment by the-liquid-metal** [comment]
No, but whenever we want, we reset all movies to the begining/end of each. Like save the playlist without save the playback position, close the playlist, and then reopen it again.

**Comment by vzhd1701** [comment]
So it's simply like a jump to the beginning?

**Comment by the-liquid-metal** [comment]
> So it's simply like a jump to the beginning?

Yes, it is.

Additionally, jump to the end functionality, with different keystroke of course. This may seem strange, but with additional backward jump, it makes a lot of sense.

**Comment by vzhd1701** [comment]
But jumping to the end right now will just loop back to the beginning or skip to next file if you loop over directory. Maybe you also need ability to stop video after it reached the end?

**Comment by the-liquid-metal** [comment]
> Maybe you also need ability to stop video after it reached the end?

Yes, if it is necessary. But if GridPlayer can achieve #19 ([Feature request]: able to take snapshot of all movies and restore to the taken state), jump to the end functionality is less importance (at least for now).

**Comment by vzhd1701** [comment]
I added percent jumping. I think jumping to 0% fills this function?

![image](https://user-images.githubusercontent.com/5212766/175576158-d70a93b5-ac11-406f-9060-a7d47b13748e.png)
